Documentation/VisualThemes.md

```diff
@@ -0,0 +1,281 @@
+
+# Visual Themes
+
+Themes allow for flexible control of UX assets in respond to various states transitions. This may involve changing a button's color, resizing an element in respond to focus, etc. The Visual Themes framework is made up of two key pieces: 1) configuration and 2) runtime engines.
```

Comment:
nit: "response"

Reply:
```suggestion
Themes allow for flexible control of UX assets in respond to various states transitions. This may involve changing a button's color, resizing an element in response to focus, etc. The Visual Themes framework is made up of two key pieces: 1) configuration and 2) runtime engines.
```